health/micro-ui/web/micro-ui-internals/packages/modules/health-dss/src/components/AdvancedMapCharts/HeatMap/Map.js

```
@@ -0,0 +1,400 @@
+                  {({ geographies }) => {
+                    return geographies.map((geo) => {                      
+                      const locationName = getTitleHeading(geo.properties?.name);
+                      const level = geo.properties?.level;
+                      const hasCoordinatesDown = geo.properties?.hasCoordinatesDown;
+                      if (filterFeature && filterFeature.finalFilter !== locationName) {
+                        return null;
+                      }
+
+                      const value = chartData?.[locationName];
+                      let cursor = "pointer",
+                        stroke = "#C6C6C6";
```

Biome (1.9.4)

[error] Declare variables separately

Unsafe fix: Break out into multiple declarations


(lint/style/useSingleVarDeclarator)